src/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
Add variables for all main values like card width, all the colors, transition time etc.